tslint.json

```diff
@@ -77,6 +77,18 @@
       "check-format",
       "allow-leading-underscore",
       "allow-pascal-case"
+    ],
+    "whitespace": [true,
+        "check-branch",
+        "check-decl",
+        "check-operator",
+        // "check-module", <- this rule is not compatible with
```

Comment:
😄 Let's face the facts. Newline is newline. Whitespace is whitespace. If newline counts as a whitespace should we treat it as a trailing whitespace as well and trigger the error here?

I think there is a room for new rule here - something like `treat-newline-as-whitespace` but otherwise i would really thing twice by making newline to be treated as whitespace.